src/openrct2/management/Research.cpp

```diff
@@ -52,7 +52,7 @@ uint8_t gResearchExpectedDay;
 std::optional<ResearchItem> gResearchNextItem;
 
 // 0x01358844[500]
-ResearchItem gResearchItems[MAX_RESEARCH_ITEMS];
+// ResearchItem gResearchItems[MAX_RESEARCH_ITEMS];
```

Comment:
This line can be removed altogether. It has been replaced with the two vectors below it.